.github/dependabot.yml

```diff
@@ -71,7 +71,9 @@ updates:
         update-types:
           - "version-update:semver-major"
           - "version-update:semver-minor"
-
+      - dependency-name: "com.github.johnrengelman.shadow"
+          update-types:
+          - "version-update:semver-major"
```

Comment:
does this need to be indented or is it already? I can't tell

Reply:
Oh yes yes, it is indented in my IDE

![image](https://github.com/aws-observability/aws-otel-test-framework/assets/41936996/f97f2468-1897-47c9-92fc-907d5310b460)

Reply:
Oh yes yes, it is indented in my IDE. double check and recommited

![image](https://github.com/aws-observability/aws-otel-test-framework/assets/41936996/f5752d7d-e522-4ec6-969a-125513a10e59)